docs/source/explanation/additional_features/auto_configuration.rst

```diff
@@ -1,4 +1,37 @@
 Auto-configuration
 ==================
 
-Coming soon...
\ No newline at end of file
+Auto-configuration for a deep learning framework means the automatic finding of the most appropriate settings for the training parameters, based on the dataset and the specific task at hand.
+Auto-configuration can help to save time, ease the process of interaction with OTX CLI and give a better baseline for the given dataset.
+
+At this end, we developed a simple auto-configuration functionality to ease the process of training and validation utilizing our framework.
+Basically, to start the training and obtain a good baseline with the best trade-off between accuracy and speed we need to pass only a dataset in the right format without specifiing anything else:
+
+.. code-block::
+
+    $ otx train --train-data-root <path_to_data_root>
+
+Moreover, our dataset can have no train/val splits at all. The Datumaro manager integrated into OTX will handle it on its own.
+It will recognize the task by analizing the dataset and if there is no splits for the validation - Datumaro will do random auto-split, memorizing this split to further use it with ``otx eval`` and ``otx optimize``.
+Currently, Datumaro auto-split feature supports 3 tasks: **multi-class classification**, **detection** and **semantic segmentation**.
+After dataset preparation the training will be started with the middle-sized template to achieve a competitive accuracy preserving fast inference.
+
+Supported dataset formats for each task:
```

Comment:
Since it is a description for auto-configuration, how about removing the datasets that can't be used for auto-configurations?

For this, I need to investigate time to analyze which dataset can be used for auto-configuration. Currently, Only 3 datasets (`Imagenet (cls)` `COCO (det)` `Cityscape (seg)`) can be auto-splitted by Datumaro

Reply:
If we have splits for other datasets and task can we use a simple auto-config?

Reply:
I mean running `otx train --train-data-roots <path_to_data_root>` with other formats if we have validation folder exactly with the same name as it should be in original dataset format

Reply:
If we don't use auto-split function, then we can use datasets below:
Classification: ImageNet
Detection: COCO, VOC, YOLO
Segmentation: Common Semantic Segmentation, VOC, Cityscape, ADE20K 2017, 2020

`otx train --train-data-roots <coco_path>`

Reply:
I moved supported dataset formats into note section. It is a good idea to give an overview about all supported datasets. We also have a note about supported formats by auto-split